contrib/codeberg-pages/template.py

```diff
@@ -0,0 +1,55 @@
+pkgname = "codeberg-pages"
+pkgver = "5.1"
+pkgrel = 0
+build_style = "go"
+make_env = {
+    "CGO_ENABLED": "1",
+}
```

Comment:
please use `env` instead of `make_env`, i intentionally specified `env` instead of `make_env` in my review comment, as the go build style doesn't use the latter at all